harness/determined/launch/autodeepspeed.py

```diff
@@ -4,21 +4,23 @@
 It launches the entrypoint script using DeepSpeed's launch process.
 """
 import argparse
+import contextlib
 import logging
 import os
 import pathlib
-import socket
+import signal
 import subprocess
 import sys
 import time
-from typing import List
+from typing import Iterator, List
 
 from deepspeed.launcher.runner import DEEPSPEED_ENVIRONMENT_NAME
 
 import determined as det
 from determined import constants
 from determined.common import api
 from determined.common.api import certs
+from determined.util import check_sshd
```

Comment:
please follow [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports).

```
from determined import util
...
util.check_sshd(...)
```